src/OpenSage.Game/Content/ShapeLoader.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using OpenSage.Data;
+using OpenSage.Data.Apt;
+using OpenSage.Gui.Apt;
+
+namespace OpenSage.Content
+{
+    internal sealed class ShapeLoader : ContentLoader<ShapeComponent>
+    {
+        public ShapeLoader(ContentManager contentManager)
+        {
+            switch (contentManager.SageGame)
+            {
+                case SageGame.CncGenerals:
+                case SageGame.CncGeneralsZeroHour:
+                    contentManager.IniDataContext.LoadIniFile(@"Data\English\HeaderTemplate.ini");
```

Comment:
Are header templates and mapped images really necesary for `.ru`? I guess this is left in from `WindowLoader`.

Reply:
Yes